tests/settings/ConfigurationGatheringTest.py

```diff
@@ -40,20 +45,19 @@ def test_gather_configuration(self):
             sections, local_bears, global_bears, targets = (
                 gather_configuration(
                     *args,
-                    arg_list=["-S",
-                              "test=5",
-                              "-c",
-                              escape(temporary, "\\"),
-                              "-s"]))
+                    arg_list=["-S", "test=5", "-c", escape(temporary, "\\"),
+                              "-s"] + self.min_args))
 
-        self.assertEqual(str(sections["default"]),
-                         "Default {config : " +
-                         repr(temporary) + ", save : 'True', test : '5'}")
+        self.assertEqual(
+            str(sections["default"]),
+            "Default {bears : 'JavaTestBear', config : " + repr(temporary) +
```

Comment:
As you are breaking blame, perhaps use `%s` or `{}` to insert `repr(temporary)` into the string.